charts/br-sisbajud/templates/topics/job.yaml

```
@@ -0,0 +1,113 @@
+#
+# STREAMING_* config is resolved from brSisbajud.extraEnvVars (wins) then
+# brSisbajud.configmap — the same idiom the migration Job uses for
+# ALLOW_INSECURE_TLS — so gitops needs no duplicated secret refs. The SASL/TLS
+# values are AVP <path:...> placeholders re-emitted as plain env, exactly as the
+# app itself consumes them.
+{{- $extraEnv := .Values.brSisbajud.extraEnvVars | default list -}}
+{{- $cfg := .Values.brSisbajud.configmap | default dict -}}
+{{- $resolved := dict -}}
+{{- range $extraEnv -}}
+{{-   if hasKey . "value" -}}
+{{-     $_ := set $resolved .name (toString .value) -}}
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)